lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/client/parameters/RootFolder.java

```diff
@@ -24,7 +24,7 @@
 /**
  * The folder where logs etc. shall be stored.
  */
-@NamedParameter(default_value = RootFolder.DEFAULT_VALUE, doc = "The folder where logs etc. shall be stored.")
+@NamedParameter(default_value = RootFolder.DEFAULT_VALUE, doc = "The folder where the logs and results are stored.")
 public final class RootFolder implements Name<String> {
   public static final String DEFAULT_VALUE = "REEF_LOCAL_RUNTIME";
```

Comment:
As for this issue, I run the task with `StandaloneRuntimeConfiguration` using the `LocalJobSubmissionHandler` since it can be reused, but the `LocalJobSubmissionHandler` gets its parameter `rootFolder` from this class. Currently, I'm struggling to find a solution to be able to put the parameter `rootFolder` from `standalone/client/parameters/RootFolder`, but currently what I can think of is to extend the LocalJobSubmissionHandler, and removing the final. Suggestions will be extremely helpful here.